mappings/cwe.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "VRT to CWE Mapping",
+  "description": "Mapping from the Vulnerability Rating Taxonomy to CWE",
+  "definitions": {
+    "MappingMetadata": {
+      "type": "object",
+      "properties": {
+        "default": { "$ref": "#/definitions/CWE" }
+      },
+      "required": ["default"]
+    },
+    "VRTid": { "type": "string", "pattern": "^[a-z_][a-z_0-9]*$" },
+    "CWE": { "type" : "array",
+      "items" : { "type": "string", "pattern": "^CWE-[0-9]*$" }
```

Comment:
is there a minimum number of entries in this array? Google suggests there are `minItems` and `uniqueItems` properties that could be added if that's relevant: http://json-schema.org/example1.html